src/test/regress/multi_1_schedule

```diff
@@ -305,6 +305,11 @@ test: global_cancel
 test: sequences_owned_by
 test: remove_coordinator
 
+# ----------
+# conversion between regular and tenant schemas
+test: citus_schema_distribute_undistribute
+# ----------
```

Comment:
Is there specific reason to no parallelize this test with some other? Maybe with schema_based_sharding.sql?